build.gradle

```
@@ -87,23 +105,27 @@ model {
                     cCompiler.args '-g'
                 }
                 cCompiler.args '-O3'
-                cCompiler.args '-fPIC'
                 cCompiler.args '-std=gnu99'
                 cCompiler.args '-I', "${Jvm.current().javaHome}/include"
 
                 // Macos
                 if (targetPlatform.operatingSystem.macOsX) {
+                    cCompiler.args '-fPIC'
                     cCompiler.args '-I', "${Jvm.current().javaHome}/include/darwin"
                     cCompiler.args '-I', ext.includePythonDir
-                    cCompiler.args '-I', ext.includNumpyDir
+                    cCompiler.args '-I', ext.includeNumpyDir
                     linker.args '-L', ext.pathLibPython
                 } else if (targetPlatform.operatingSystem.linux) { // linux
+                    cCompiler.args '-fPIC'
                     cCompiler.args '-I', "${Jvm.current().javaHome}/include/linux"
                     cCompiler.args '-I', ext.includePythonDir
-                    cCompiler.args '-I', ext.includNumpyDir
-                    cCompiler.args '-L', ext.libPython
+                    cCompiler.args '-I', ext.includeNumpyDir
+                    linker.args '-L', ext.pathLibPython
                 } else { // win
```

Let's maybe add `else if(windows)` there too, and throw `Unsupported platform` exception in `else` below?